src/index.js

```diff
@@ -101,6 +101,7 @@ const formatOptions = ({steps, ...otherOptions}) => ({
 
 
 Onfido.init = (opts) => {
+  console.log("onfido_sdk_version", process.env.SDK_VERSION)
```

Comment:
OK makes sense. I would leave it - putting it in the Onfido object just makes it harder to access.